src/ORM/Sluggable/SluggableSubscriber.php

```diff
@@ -93,4 +102,26 @@ private function isSluggable(ClassMetadata $classMetadata)
             $this->isRecursive
         );
     }
+
+    /**
+     * @param $entity
+     * @param EntityManager $em
+     */
+    private function generateUniqueSlugFor($entity, EntityManager $em)
+    {
+        $classMetadata = $em->getClassMetadata(get_class($entity));
```

Comment:
why using classMetadata?

You can directly do : `$em->getRepository(get_class($entity))`